spec/controllers/users/sessions_controller_spec.rb

```diff
@@ -191,6 +191,7 @@
     it 'tracks the successful authentication for existing user' do
       user = create(:user, :signed_up)
       subject.session['user_return_to'] = 'http://example.com'
+      email = user.email.upcase
```

Comment:
It wasn't obvious to me why the existing `post` line was calling `upcase` here, but doing the same thing in the assertion I added would be even less intuitive, so I pulled this out into a variable to make sure they're the same.

On closer examination, I don't actually think the `.upcase` was ever necessary, if we wanted to just remove that?